tests/data_context/store/test_ge_cloud_store_backend.py

```diff
@@ -30,7 +30,7 @@ def construct_ge_cloud_store_backend(
     ge_cloud_access_token: str,
 ) -> Callable[[GeCloudRESTResource], GeCloudStoreBackend]:
     def _closure(resource_type: GeCloudRESTResource) -> GeCloudStoreBackend:
-        ge_cloud_base_url = "https://app.greatexpectations.io/"
+        ge_cloud_base_url = "https://api.greatexpectations.io/"
```

Comment:
Can we reference a constant here and elsewhere in the file to avoid duplicating the URL?

Reply:
Yes, absolutely -- I just did a replace all, but will try to clean up.